modules/puzzles/util.py

```diff
@@ -77,3 +78,31 @@ def remove_duplicate_locations(locations):
             results.append(l)
             storedLocations.append(normalizedL)
     return results
+
+
+def brute_force_coordinates(digits_list):
+    # Add token to seperate latitude and longitude later
+    digits_list.append(',')
+    perms = [''.join(x) for x in list(map(list, permutations(digits_list, len(digits_list))))]
+
+    coordinates_list = []
+    for perm in perms:
+        lat, long = perm.split(',')
```

Comment:
`long` is een keyword in python2, en heeft dus in veel IDEs een speciale kleur. (Zie bijvoorbeeld de github code preview) Ik zou liever de variable naam `lon` of `longitude` zien.

Reply:
Goed punt, had ik kunnen zien aankomen. Fixed